Synthesis failed for java-analytics-admin

Hello! Autosynth couldn't regenerate java-analytics-admin. :broken_heart:

Here's the output from running `synth.py`:

```
Running: git log -1 --pretty=%at e28aa8570dc86136fe677202c976f4ee00d77f61
2021-02-11 15:03:05,902 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6de7abea4c424025bfcf996ef40c96e53c070100
2021-02-11 15:03:05,905 autosynth [DEBUG] > Running: git log -1 --pretty=%at 614b4fc8969008b086fbd6d3d99dfdb6fcf0a4a1
2021-02-11 15:03:05,908 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8761f14a2a2a2cde7e44c9bf8a5e901273a71812
2021-02-11 15:03:05,910 autosynth [DEBUG] > Running: git log -1 --pretty=%at 66e0de02e649c5b90f873836bdbc97585d91ed46
2021-02-11 15:03:05,913 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd89b0cc1bffc39d1fb5c91c6c2bb3532f237792
2021-02-11 15:03:05,916 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1e0c7413684ca6f6322620ecfc0d3e0352933dc1
2021-02-11 15:03:05,919 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-11 15:03:05,921 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-11 15:03:05,924 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-11 15:03:05,927 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-11 15:03:05,932 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-11 15:03:05,955 autosynth [DEBUG] > Running: git checkout f3a60f63c13fb434745ea59b990a82d6ffc803b5
Note: checking out 'f3a60f63c13fb434745ea59b990a82d6ffc803b5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at f3a60f63 docs: Update rules for currency_code in budget_amount.
2021-02-11 15:03:05,969 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-11 15:03:05,975 autosynth [DEBUG] > Running: git branch -f autosynth-90
2021-02-11 15:03:05,978 autosynth [DEBUG] > Running: git checkout autosynth-90
Switched to branch 'autosynth-90'
2021-02-11 15:03:05,982 autosynth [INFO] > Running synthtool
2021-02-11 15:03:05,982 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-11 15:03:05,983 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/90/sponge_log.log
2021-02-11 15:03:05,984 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-11 15:03:06,219 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-90
nothing to commit, working tree clean
2021-02-11 15:03:06,365 synthtool [DEBUG] > Ensuring dependencies.
2021-02-11 15:03:06,375 synthtool [DEBUG] > Cloning googleapis.
2021-02-11 15:03:06,376 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-11 15:03:06,380 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-11 15:03:06,384 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-11 15:07:02,914 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpezn7n42u.
2021-02-11 15:07:02,916 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpezn7n42u/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-11 15:07:02,920 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpezn7n42u/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpezn7n42u/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-11 15:07:02,921 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpezn7n42u/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpezn7n42u/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-11 15:07:02,975 autosynth [ERROR] > Synthesis failed
2021-02-11 15:07:02,975 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-11 15:07:02,982 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-11 15:07:02,987 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/082d9d81-121c-4af4-9cc3-cc51dcb2053d/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
ynth [DEBUG] > Running: git log -1 --pretty=%at 6de7abea4c424025bfcf996ef40c96e53c070100
2021-02-12 02:02:48,823 autosynth [DEBUG] > Running: git log -1 --pretty=%at 614b4fc8969008b086fbd6d3d99dfdb6fcf0a4a1
2021-02-12 02:02:48,827 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8761f14a2a2a2cde7e44c9bf8a5e901273a71812
2021-02-12 02:02:48,831 autosynth [DEBUG] > Running: git log -1 --pretty=%at 66e0de02e649c5b90f873836bdbc97585d91ed46
2021-02-12 02:02:48,835 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd89b0cc1bffc39d1fb5c91c6c2bb3532f237792
2021-02-12 02:02:48,839 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1e0c7413684ca6f6322620ecfc0d3e0352933dc1
2021-02-12 02:02:48,843 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-12 02:02:48,847 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-12 02:02:48,850 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-12 02:02:48,854 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-12 02:02:48,858 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-12 02:02:48,865 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-12 02:02:48,890 autosynth [DEBUG] > Running: git checkout c059c46957b0eb83078e61b4a5170c3d09ea96e6
Note: checking out 'c059c46957b0eb83078e61b4a5170c3d09ea96e6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at c059c469 chore: Operations proto comment cleanup
2021-02-12 02:02:48,908 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-12 02:02:48,915 autosynth [DEBUG] > Running: git branch -f autosynth-91
2021-02-12 02:02:48,919 autosynth [DEBUG] > Running: git checkout autosynth-91
Switched to branch 'autosynth-91'
2021-02-12 02:02:48,925 autosynth [INFO] > Running synthtool
2021-02-12 02:02:48,925 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-12 02:02:48,925 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/91/sponge_log.log
2021-02-12 02:02:48,928 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-12 02:02:49,181 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-91
nothing to commit, working tree clean
2021-02-12 02:02:49,341 synthtool [DEBUG] > Ensuring dependencies.
2021-02-12 02:02:49,355 synthtool [DEBUG] > Cloning googleapis.
2021-02-12 02:02:49,357 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-12 02:02:49,362 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-12 02:02:49,367 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-12 02:06:40,147 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpmqianxe3.
2021-02-12 02:06:40,149 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpmqianxe3/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-12 02:06:40,153 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpmqianxe3/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpmqianxe3/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-12 02:06:40,154 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpmqianxe3/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpmqianxe3/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-12 02:06:40,220 autosynth [ERROR] > Synthesis failed
2021-02-12 02:06:40,221 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-12 02:06:40,228 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-12 02:06:40,234 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/68f2f6b8-7431-47a6-a634-59c5352f7382/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
G] > Running: git log -1 --pretty=%at 1e0c7413684ca6f6322620ecfc0d3e0352933dc1
2021-02-12 15:03:53,954 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-12 15:03:53,958 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-12 15:03:53,961 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-12 15:03:53,965 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-12 15:03:53,968 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-12 15:03:53,972 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-12 15:03:53,975 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-12 15:03:53,978 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-12 15:03:53,982 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-12 15:03:53,985 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-12 15:03:53,991 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-12 15:03:54,016 autosynth [DEBUG] > Running: git checkout 178d16d9f99ee85ffa65dff304f40324c35d0668
Note: checking out '178d16d9f99ee85ffa65dff304f40324c35d0668'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 178d16d9 chore(bazel): update Go microgenerator to v0.18.1
2021-02-12 15:03:54,032 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-12 15:03:54,039 autosynth [DEBUG] > Running: git branch -f autosynth-96
2021-02-12 15:03:54,042 autosynth [DEBUG] > Running: git checkout autosynth-96
Switched to branch 'autosynth-96'
2021-02-12 15:03:54,048 autosynth [INFO] > Running synthtool
2021-02-12 15:03:54,048 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-12 15:03:54,048 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/96/sponge_log.log
2021-02-12 15:03:54,050 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-12 15:03:54,289 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-96
nothing to commit, working tree clean
2021-02-12 15:03:54,434 synthtool [DEBUG] > Ensuring dependencies.
2021-02-12 15:03:54,445 synthtool [DEBUG] > Cloning googleapis.
2021-02-12 15:03:54,446 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-12 15:03:54,450 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-12 15:03:54,454 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-12 15:07:47,322 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpz_gywm6j.
2021-02-12 15:07:47,324 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpz_gywm6j/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-12 15:07:47,328 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpz_gywm6j/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpz_gywm6j/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-12 15:07:47,328 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpz_gywm6j/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpz_gywm6j/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-12 15:07:47,398 autosynth [ERROR] > Synthesis failed
2021-02-12 15:07:47,399 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-12 15:07:47,409 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-12 15:07:47,415 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5e3736d6-6ebd-480c-bf62-78413733613e/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-13 02:01:03,270 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-13 02:01:03,274 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-13 02:01:03,277 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-13 02:01:03,280 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-13 02:01:03,283 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-13 02:01:03,286 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-13 02:01:03,289 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-13 02:01:03,293 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-13 02:01:03,296 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-13 02:01:03,299 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-13 02:01:03,304 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-13 02:01:03,328 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-13 02:01:03,343 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-13 02:01:03,349 autosynth [DEBUG] > Running: git branch -f autosynth-97
2021-02-13 02:01:03,352 autosynth [DEBUG] > Running: git checkout autosynth-97
Switched to branch 'autosynth-97'
2021-02-13 02:01:03,357 autosynth [INFO] > Running synthtool
2021-02-13 02:01:03,357 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-13 02:01:03,358 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/97/sponge_log.log
2021-02-13 02:01:03,359 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-13 02:01:03,594 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-97
nothing to commit, working tree clean
2021-02-13 02:01:03,747 synthtool [DEBUG] > Ensuring dependencies.
2021-02-13 02:01:03,759 synthtool [DEBUG] > Cloning googleapis.
2021-02-13 02:01:03,760 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-13 02:01:03,765 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-13 02:01:03,769 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-13 02:04:38,111 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp51aainjf.
2021-02-13 02:04:38,113 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp51aainjf/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-13 02:04:38,116 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp51aainjf/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmp51aainjf/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-13 02:04:38,117 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp51aainjf/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmp51aainjf/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-13 02:04:38,185 autosynth [ERROR] > Synthesis failed
2021-02-13 02:04:38,185 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-13 02:04:38,191 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-13 02:04:38,197 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/562f66c4-78e1-4e49-814f-ad1fe2f28808/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-13 15:02:20,030 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-13 15:02:20,033 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-13 15:02:20,037 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-13 15:02:20,040 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-13 15:02:20,043 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-13 15:02:20,046 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-13 15:02:20,049 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-13 15:02:20,053 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-13 15:02:20,056 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-13 15:02:20,059 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-13 15:02:20,065 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-13 15:02:20,090 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-13 15:02:20,105 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-13 15:02:20,111 autosynth [DEBUG] > Running: git branch -f autosynth-97
2021-02-13 15:02:20,115 autosynth [DEBUG] > Running: git checkout autosynth-97
Switched to branch 'autosynth-97'
2021-02-13 15:02:20,119 autosynth [INFO] > Running synthtool
2021-02-13 15:02:20,120 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-13 15:02:20,120 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/97/sponge_log.log
2021-02-13 15:02:20,122 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-13 15:02:20,373 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-97
nothing to commit, working tree clean
2021-02-13 15:02:20,526 synthtool [DEBUG] > Ensuring dependencies.
2021-02-13 15:02:20,537 synthtool [DEBUG] > Cloning googleapis.
2021-02-13 15:02:20,538 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-13 15:02:20,543 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-13 15:02:20,547 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-13 15:06:14,100 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp1hgfdn6_.
2021-02-13 15:06:14,102 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp1hgfdn6_/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-13 15:06:14,106 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp1hgfdn6_/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmp1hgfdn6_/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-13 15:06:14,106 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmp1hgfdn6_/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmp1hgfdn6_/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-13 15:06:14,161 autosynth [ERROR] > Synthesis failed
2021-02-13 15:06:14,162 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-13 15:06:14,167 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-13 15:06:14,172 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2f1022c1-09f4-4cc1-bac8-062aa7db90e2/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-14 02:00:26,806 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-14 02:00:26,809 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-14 02:00:26,811 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-14 02:00:26,814 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-14 02:00:26,817 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-14 02:00:26,820 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-14 02:00:26,823 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-14 02:00:26,826 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-14 02:00:26,829 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-14 02:00:26,832 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-14 02:00:26,837 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-14 02:00:26,860 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-14 02:00:26,875 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-14 02:00:26,880 autosynth [DEBUG] > Running: git branch -f autosynth-97
2021-02-14 02:00:26,883 autosynth [DEBUG] > Running: git checkout autosynth-97
Switched to branch 'autosynth-97'
2021-02-14 02:00:26,888 autosynth [INFO] > Running synthtool
2021-02-14 02:00:26,888 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-14 02:00:26,888 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/97/sponge_log.log
2021-02-14 02:00:26,890 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-14 02:00:27,110 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-97
nothing to commit, working tree clean
2021-02-14 02:00:27,250 synthtool [DEBUG] > Ensuring dependencies.
2021-02-14 02:00:27,260 synthtool [DEBUG] > Cloning googleapis.
2021-02-14 02:00:27,261 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-14 02:00:27,266 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-14 02:00:27,269 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-14 02:03:53,605 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpetxflldr.
2021-02-14 02:03:53,606 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpetxflldr/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-14 02:03:53,610 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpetxflldr/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpetxflldr/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-14 02:03:53,610 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpetxflldr/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpetxflldr/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-14 02:03:53,673 autosynth [ERROR] > Synthesis failed
2021-02-14 02:03:53,673 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-14 02:03:53,679 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-14 02:03:53,684 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/31fd9052-93c2-4dbc-ae81-21c9951f8f8f/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-14 15:00:55,618 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-14 15:00:55,621 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-14 15:00:55,624 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-14 15:00:55,627 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-14 15:00:55,630 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-14 15:00:55,633 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-14 15:00:55,636 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-14 15:00:55,639 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-14 15:00:55,642 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-14 15:00:55,644 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-14 15:00:55,650 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-14 15:00:55,654 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-14 15:00:55,733 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-14 15:00:55,739 autosynth [DEBUG] > Running: git branch -f autosynth-97
2021-02-14 15:00:55,743 autosynth [DEBUG] > Running: git checkout autosynth-97
Switched to branch 'autosynth-97'
2021-02-14 15:00:55,748 autosynth [INFO] > Running synthtool
2021-02-14 15:00:55,748 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-14 15:00:55,748 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/97/sponge_log.log
2021-02-14 15:00:55,750 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-14 15:00:55,984 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-97
nothing to commit, working tree clean
2021-02-14 15:00:56,129 synthtool [DEBUG] > Ensuring dependencies.
2021-02-14 15:00:56,140 synthtool [DEBUG] > Cloning googleapis.
2021-02-14 15:00:56,141 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-14 15:00:56,145 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-14 15:00:56,149 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-14 15:04:32,376 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpnt354gmr.
2021-02-14 15:04:32,378 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpnt354gmr/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-14 15:04:32,382 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpnt354gmr/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpnt354gmr/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-14 15:04:32,382 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpnt354gmr/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpnt354gmr/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-14 15:04:32,457 autosynth [ERROR] > Synthesis failed
2021-02-14 15:04:32,457 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-14 15:04:32,467 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-14 15:04:32,472 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9a734575-adfb-43a1-acd6-2d5255cd7476/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-15 02:01:57,655 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-15 02:01:57,659 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-15 02:01:57,663 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-15 02:01:57,666 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-15 02:01:57,670 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-15 02:01:57,673 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-15 02:01:57,677 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-15 02:01:57,680 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-15 02:01:57,684 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-15 02:01:57,688 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-15 02:01:57,694 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-15 02:01:57,700 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-15 02:01:57,777 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-15 02:01:57,783 autosynth [DEBUG] > Running: git branch -f autosynth-97
2021-02-15 02:01:57,788 autosynth [DEBUG] > Running: git checkout autosynth-97
Switched to branch 'autosynth-97'
2021-02-15 02:01:57,794 autosynth [INFO] > Running synthtool
2021-02-15 02:01:57,794 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-15 02:01:57,794 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/97/sponge_log.log
2021-02-15 02:01:57,797 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-15 02:01:58,048 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-97
nothing to commit, working tree clean
2021-02-15 02:01:58,202 synthtool [DEBUG] > Ensuring dependencies.
2021-02-15 02:01:58,214 synthtool [DEBUG] > Cloning googleapis.
2021-02-15 02:01:58,215 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-15 02:01:58,220 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-15 02:01:58,225 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-15 02:05:48,391 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpuae39oec.
2021-02-15 02:05:48,394 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpuae39oec/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-15 02:05:48,402 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpuae39oec/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpuae39oec/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-15 02:05:48,403 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpuae39oec/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpuae39oec/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-15 02:05:48,465 autosynth [ERROR] > Synthesis failed
2021-02-15 02:05:48,465 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-15 02:05:48,472 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-15 02:05:48,479 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f88c5505-c115-4c44-a4df-7518606354ad/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-15 15:02:11,433 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-15 15:02:11,437 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-15 15:02:11,441 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-15 15:02:11,445 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-15 15:02:11,449 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-15 15:02:11,453 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-15 15:02:11,457 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-15 15:02:11,461 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-15 15:02:11,464 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-15 15:02:11,468 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-15 15:02:11,475 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-15 15:02:11,500 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-15 15:02:11,533 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-15 15:02:11,540 autosynth [DEBUG] > Running: git branch -f autosynth-97
2021-02-15 15:02:11,544 autosynth [DEBUG] > Running: git checkout autosynth-97
Switched to branch 'autosynth-97'
2021-02-15 15:02:11,550 autosynth [INFO] > Running synthtool
2021-02-15 15:02:11,551 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-15 15:02:11,551 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/97/sponge_log.log
2021-02-15 15:02:11,553 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-15 15:02:11,796 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-97
nothing to commit, working tree clean
2021-02-15 15:02:11,952 synthtool [DEBUG] > Ensuring dependencies.
2021-02-15 15:02:11,965 synthtool [DEBUG] > Cloning googleapis.
2021-02-15 15:02:11,966 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-15 15:02:11,971 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-15 15:02:11,976 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-15 15:05:57,849 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpgnyhap20.
2021-02-15 15:05:57,851 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpgnyhap20/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-15 15:05:57,854 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpgnyhap20/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpgnyhap20/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-15 15:05:57,855 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpgnyhap20/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpgnyhap20/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-15 15:05:57,913 autosynth [ERROR] > Synthesis failed
2021-02-15 15:05:57,913 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-15 15:05:57,920 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-15 15:05:57,926 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/054c0893-56c7-417a-9b9b-8974285e2450/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

Autosynth is still having trouble generating java-analytics-admin. :sob:

Here's the output from running `synth.py`:

```
git log -1 --pretty=%at 3f914b89ae1ef459955f470f9ccb498314ad28d5
2021-02-16 02:00:20,977 autosynth [DEBUG] > Running: git log -1 --pretty=%at f3a60f63c13fb434745ea59b990a82d6ffc803b5
2021-02-16 02:00:20,980 autosynth [DEBUG] > Running: git log -1 --pretty=%at c059c46957b0eb83078e61b4a5170c3d09ea96e6
2021-02-16 02:00:20,983 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5a11760c9dcdf63deba111e44bc12df5a4b5cd3a
2021-02-16 02:00:20,986 autosynth [DEBUG] > Running: git log -1 --pretty=%at f9cd594c912a2ff456cc2b33f54dcc238133bb0c
2021-02-16 02:00:20,989 autosynth [DEBUG] > Running: git log -1 --pretty=%at f235b894ae127e6239b99ee1d7c7755f686cd76f
2021-02-16 02:00:20,992 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b2ab49ed6df31252afdede55280f97e1a745962
2021-02-16 02:00:20,995 autosynth [DEBUG] > Running: git log -1 --pretty=%at 178d16d9f99ee85ffa65dff304f40324c35d0668
2021-02-16 02:00:20,998 autosynth [DEBUG] > Running: git log -1 --pretty=%at 78f5365656fda2855cf0546080da559d6c2ed3ff
2021-02-16 02:00:21,001 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2414b817065726eae0bc525346c7e874f969369d
2021-02-16 02:00:21,004 autosynth [DEBUG] > Running: git log -1 --pretty=%at b32c87d2c087fb2a841acc623ab540105de821af
2021-02-16 02:00:21,009 autosynth [DEBUG] > Running: git checkout 0d0527a1a7faccde4cab9ba9f310cd0b602bc501
Note: checking out '0d0527a1a7faccde4cab9ba9f310cd0b602bc501'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-16 02:00:21,032 autosynth [DEBUG] > Running: git checkout 78f5365656fda2855cf0546080da559d6c2ed3ff
Note: checking out '78f5365656fda2855cf0546080da559d6c2ed3ff'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 78f53656 build: add BUILD files to generate client library for orgpolicy
2021-02-16 02:00:21,047 autosynth [DEBUG] > Running: git checkout b32c87d2c087fb2a841acc623ab540105de821af
Note: checking out 'b32c87d2c087fb2a841acc623ab540105de821af'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b32c87d feat: add googleapis-gen as a tracked source (#944)
2021-02-16 02:00:21,052 autosynth [DEBUG] > Running: git branch -f autosynth-97
2021-02-16 02:00:21,055 autosynth [DEBUG] > Running: git checkout autosynth-97
Switched to branch 'autosynth-97'
2021-02-16 02:00:21,059 autosynth [INFO] > Running synthtool
2021-02-16 02:00:21,060 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-02-16 02:00:21,060 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-analytics-admin/97/sponge_log.log
2021-02-16 02:00:21,061 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-02-16 02:00:21,286 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py.
On branch autosynth-97
nothing to commit, working tree clean
2021-02-16 02:00:21,424 synthtool [DEBUG] > Ensuring dependencies.
2021-02-16 02:00:21,434 synthtool [DEBUG] > Cloning googleapis.
2021-02-16 02:00:21,435 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2021-02-16 02:00:21,439 synthtool [DEBUG] > Which version of bazel will I run?
bazel 3.0.0
2021-02-16 02:00:21,443 synthtool [DEBUG] > Generating code for: //google/analytics/admin/v1alpha:google-analytics-admin-v1alpha-java.
2021-02-16 02:03:46,981 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpw58lx_7i.
2021-02-16 02:03:46,982 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpw58lx_7i/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern // Generated by the protocol buffer compiler.  DO NOT EDIT!, maybe replacement is no longer needed?
2021-02-16 02:03:46,986 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpw58lx_7i/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Name.java'), PosixPath('/tmpfs/tmp/tmpw58lx_7i/google-cloud-analytics-admin-v1alpha-java/proto-google-analytics-admin-v1alpha-java/src/**/*Names.java')] for pattern /\*
 \* Copyright \d{4} Google LLC
 \*
 \* Licensed under the Apache License, Version 2.0 \(the "License"\); you may not use this file except
 \* in compliance with the License. You may obtain a copy of the License at
 \*
 \* http://www.apache.org/licenses/LICENSE-2.0
 \*
 \* Unless required by applicable law or agreed to in writing, software distributed under the License
 \* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 \* or implied. See the License for the specific language governing permissions and limitations under
 \* the License.
 \*/
, maybe replacement is no longer needed?
2021-02-16 02:03:46,986 synthtool [WARNING] > No replacements made in [PosixPath('/tmpfs/tmp/tmpw58lx_7i/google-cloud-analytics-admin-v1alpha-java/grpc-google-analytics-admin-v1alpha-java/src/**/*.java')] for pattern ^package (.*);, maybe replacement is no longer needed?
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-analytics-admin/synth.py", line 30, in <module>
    cloud_api=False,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 317, in bazel_library
    diregapic=diregapic,
  File "/tmpfs/src/github/synthtool/synthtool/languages/java.py", line 199, in _common_generation
    required=True,
  File "/tmpfs/src/github/synthtool/synthtool/transforms.py", line 214, in move
    f"No files in sources {sources} were copied. Does the source "
synthtool.transforms.MissingSourceError: No files in sources [PosixPath('/tmpfs/tmp/tmpw58lx_7i/google-cloud-analytics-admin-v1alpha-java/gapic-google-analytics-admin-v1alpha-java/src')] were copied. Does the source contain files?
2021-02-16 02:03:47,041 autosynth [ERROR] > Synthesis failed
2021-02-16 02:03:47,041 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 0d0527a build(java): update autorelease script (#153)
2021-02-16 02:03:47,047 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2021-02-16 02:03:47,052 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0932c387-5458-427c-821a-c508d306b680/targets/github%2Fsynthtool;config=default/tests;query=java-analytics-admin;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -14,20 +14,17 @@
 
 """This script is used to synthesize generated parts of this library."""
 
-import synthtool as s
-import synthtool.gcp as gcp
 import synthtool.languages.java as java
 
 service = 'analytics-admin'
 versions = ['v1alpha']
 
 for version in versions:
-  java.bazel_library(
-      service=service,
-      version=version,
-      proto_path=f'google/analytics/admin/{version}',
-      bazel_target=f'//google/analytics/admin/{version}:google-{service}-{version}-java',
-      cloud_api=False,
+  java.pregenerated_library(
+    service=service,
+    version=version,
+    path=f'google/analytics/admin/{version}',
+    cloud_api=False,
   )
 
 java.common_templates()

